pymchelper/readers/shieldhit/reader_bdo2019.py

```diff
@@ -0,0 +1,126 @@
+import logging
+
+import numpy as np
+
+from pymchelper.estimator import Page, MeshAxis
+from pymchelper.readers.shieldhit.reader_base import SHReader, read_next_token
+from pymchelper.readers.shieldhit.binary_spec import SHBDOTagID, detector_name_from_bdotag, page_name_from_bdotag, \
+    unit_name_from_unit_id
+from pymchelper.shieldhit.detector.detector_type import SHDetType
+from pymchelper.shieldhit.detector.estimator_type import SHGeoType
+
+logger = logging.getLogger(__name__)
+
+
+class SHReaderBDO2019(SHReader):
```

Comment:
currently the differential units are not stored in SHBDO_PAG_DATA_UNITS, but in SHBDO_PAG_DIF_UNITS.
Because of that, printed units will appear separated with ";" instead of "/".

But I think it is better to change SH12A, since your solution seems intuitively better.